js/main.js

```diff
@@ -43,3 +51,124 @@ hamBttn.addEventListener('click', () => {
     });
   }
 });
+
+// Projects popup
+const popupDivBG = document.createElement('div');
+const popupDiv = document.createElement('div');
+const projectBttns = document.querySelectorAll('.popup');
+const projectDiv = document.createElement('div');
+const myProjects = [{
+  name: 'Tonic',
+  imgUrl: './Images/pages/SnapshootPortfolio.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+{
+  name: 'Multi-post Stories',
+  imgUrl: './Images/pages/SnapshootPortfolio1.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+{
+  name: 'Tonic',
+  imgUrl: './Images/pages/SnapshootPortfolio2.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+{
+  name: 'Multi-post Stories',
+  imgUrl: './Images/pages/SnapshootPortfolio3.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+];
+
+function createWorksSection() {
+  if (myProjects.length !== 0) {
+    const myWorksSection = document.querySelector('.myWorksSection');
+    for (let i = 0; i < myProjects.length; i += 1) {
+      let technologies = '';
+      for (let j = 0; j < myProjects[i].techs.length; j += 1) {
+        technologies += `<li>${myProjects[i].techs[j]}</li>`;
+      }
+      myWorksSection.innerHTML += '<li class="tonic">'
+        + `<img src="${myProjects[i].imgUrl}" alt="project background image" class="image-responsive">`
+        + '<div>'
+        + `<h3>${myProjects[i].name}</h3>`
+        + '<ul class="work-info">'
+        + '<li class="canopy">CANOPY</li>'
+        + '<li><span class="span"></span></li>'
+        + '<li class="canopy1">Back End Dev</li>'
+        + '<li><span class="span"></span></li>'
+        + '<li class="canopy1">2015</li>'
+        + '</ul>'
+        + `<p class="despription">${myProjects[i].description}</p>`
+        + `<ul class="tech-info">${technologies}</ul>`
+        + `<button class="btn popup" type="button">${myProjects[i].button}</button>`
+        + '</div>'
+        + '</li>';
+    }
+  }
+}
+
+createWorksSection();
+
+function addEventListenerList() {
+  for (let i = 0; i < projectBttns.length; i += 1) {
+    projectBttns[i].addEventListener('click', () => {
+      popupDiv.id = 'overlay-popup';
+      popupDivBG.id = 'overlay-popup-bg';
+      projectDiv.id = 'overlay-project';
```

Comment:
- [ ] Kindly use `arrow functions` over the `function` keyword. Refer to the previous requirement i have given a detailed explanation of this